flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1341,26 +1365,26 @@ def test_created_is_listed(self):
 
     def test_listed_volume_attributes(self):
         """
-        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the same
-        dataset_id and size as was passed to ``create_volume``.
+        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the
+        same dataset_id and (maybe over-allocated size) as was passed
+        to ``create_volume``.
         """
         expected_dataset_id = uuid4()
+        requested_size = REALISTIC_BLOCKDEVICE_SIZE
```

Comment:
I added a `minimum_allocatable_size` variable to the test case factory and set it to `1GiB` on AWS and OpenStack and `100GiB` on Rackspace based on the supplied CLOUD_PROVIDER environment variable.